rust/connlib/tunnel/src/client.rs

```diff
@@ -793,17 +793,28 @@ impl ClientState {
         let maybe_dns_resource_id = self
             .stub_resolver
             .resolve_resource_by_ip(&destination)
-            .filter(|resource| self.is_resource_enabled(resource));
+            .filter(|resource| self.is_resource_enabled(resource))
+            .inspect(
+                |resource| tracing::trace!(%destination, %resource, "Packet for DNS resource"),
```

Comment:
Should we `cfg(test)` this? We sometimes run production code with `firezone_tunnel=trace` so this will be very noisy!

Reply:
what about a `"tunnel_test"` target?

Reply:
Yeah that would work too. Or maybe `tunnel_test_coverage` to be super explicit?